plugins/foreman_discovery/18.0/index.md

```diff
@@ -0,0 +1,953 @@
+latest stable image. For this, re-run the installer with the following option:
+
+    # foreman-installer \
+      --enable-foreman-proxy-plugin-discovery \
+      --foreman-proxy-plugin-discovery-install-images=true
+
+The command installs latest available Foreman Discovery Image. If that's not the version required (see the compatibility table above), explicitly request to download specific version:
+
+    # foreman-installer \
+      --enable-foreman-proxy-plugin-discovery \
+      --foreman-proxy-plugin-discovery-install-images=true \
+      --foreman-proxy-plugin-discovery-source-url=http://downloads.theforeman.org/discovery/releases/{{page.imgver}}/
+
+Tip: It is possible to install both Discovery plugin and image in one
+installer run by providing both the options.
+
+Important note: Executing foreman-installer will re-deploy all foreman-related
+configuration files. In case some changes have been made, proceed with manual
+download described below.
+
+### 2.3.2 Manual download
+
+Images are available [for direct
+download](http://downloads.theforeman.org/discovery/)
+To download the latest release to the expected location, do the following on
+Fedora and Red Hat systems:
+
+    # wget http://downloads.theforeman.org/discovery/releases/{{page.imgver}}/fdi-image-latest.tar \
+      -O - | tar x --overwrite -C /var/lib/tftpboot/boot
+
+Check the version number in the url against the table above. Some image-plugin
+combinations are not compatible.
+
+On Debian systems, use `/srv/tftp/boot` instead of `/var/tftpboot/boot`.
+
+### 2.3.3 Verify signatures and checksums
+
+To verify individual ISO/tar files, import our Discovery Plugin key
+[7E81E7B0](http://keys.fedoraproject.org:11371/pks/lookup?search=0x7E81E7B0&op=get)
+and verify individual files:
+
+    # wget 'http://keys.fedoraproject.org:11371/pks/lookup?search=0x7E81E7B0&op=get' -O- | gpg --import
+    # gpg --verify --multifile *.asc
+    gpg: Signature made Fri 06 Nov 2015 09:42:55 AM UTC using RSA key ID 7E81E7B0
+    gpg: Good signature from "Foreman Discovery <foreman-xxx@googlegroups.com>"
+
+To verify extracted init RAM disk and linux kernel do:
+
+    # cat /var/lib/tftpboot/boot/fdi-image/SHA256SUM
+    beb3cfba7d9fb9d71481c0c8f... initrd0.img
+    f03bce150d2473a28697d2988... vmlinuz0
+    # sha256sum /var/lib/tftpboot/boot/fdi-image/*
+
+On Debian systems, use `/srv/tftp/boot` instead of `/var/tftpboot/boot`.
+
+### 2.3.4 Building own image
+
+To build a discovery image, please visit the foreman-discovery-image git
+[repository](https://github.com/theforeman/foreman-discovery-image) and find
+the README for further instructions.
+
+## 2.4 Hammer CLI Foreman Discovery
+
+### 2.4.1 Installation
+
+Before installing make sure you have installed the
+[hammer_cli_foreman plugin](https://github.com/theforeman/hammer-cli/blob/master/doc/installation.md),
+
+On Red Hat compatible systems:
+
+```yum install tfm-rubygem-hammer_cli_foreman_discovery```
```

Comment:
if you want tripple-backticks, you want the rest on a new line, otherwise it will render badly.